cts/cli/cibadmin_invalid_schema.xml

```diff
@@ -0,0 +1,24 @@
+<cib crm_feature_set="3.19.5" validate-with="pacemaker-1.3" epoch="2" num_updates="0" admin_epoch="0">
```

Comment:
pacemaker-1.3 is a valid schema - if the idea here is to test a schema version that doesn't exist, you should change this to something that's not in `xml/pacemaker-*`.  Maybe 1.4, or 2.999, or something like that.